src/lua/utils.h

```diff
@@ -249,6 +249,14 @@ luaL_maplen(struct lua_State *L, int idx)
 
 extern int luaL_nil_ref;
 
+/**
+ * Fill a table on top of the Lua stack with functions.
+ *
+ * Leave the table on top of the stack.
```

Comment:
Don't get this comment. Am I right that this is the same table, that is filled?

Reply:
Yes. In Lua Reference manual notation it means `[0, 0]`.

Reply:
> Am I right that this is the same table, that is filled?

Yes.

> May be it is better to drop the second sentence, or specify that this is the filled table.

I've rewrote it a bit:

```c
/**   
 * Fill a table on top of the Lua stack with functions.
 *
 * The Lua stack remains unchanged after calling this function.
 */
```

Reply:
Nice, thanks!